tests/php/test-class-amp-tiktok-embed-handler.php

```diff
@@ -5,16 +5,23 @@
  * @package AMP.
  */
 
+use AmpProject\AmpWP\Tests\WithoutBlockPreRendering;
+
 /**
  * Class Test_AMP_TikTok_Embed_Handler
  */
 class Test_AMP_TikTok_Embed_Handler extends WP_UnitTestCase {
 
+	use WithoutBlockPreRendering {
+		setUp as public prevent_block_pre_render;
+	}
+
 	/**
 	 * Set up.
 	 */
 	public function setUp() {
 		parent::setUp();
+		$this->prevent_block_pre_render();
```

Comment:
This calls `parent::setUp()` twice, once here, and once in `WithoutBlockPreRendering ::setUp` (which is mapped to `$this->prevent_block_pre_render();` here).